pootle/apps/accounts/management/commands/purge_user.py

```diff
@@ -11,9 +11,8 @@
 
 
 class Command(UserCommand):
-    args = "user"
     help = "Delete user and all related objects"
 
-    def handle(self, *args, **kwargs):
-        super(Command, self).handle(*args, **kwargs)
-        self.get_user(args[0]).delete(purge=True)
+    def handle(self, **options):
+        for user in options['user']:
```

Comment:
Does this now accept multiple username arguments? If so, docs need to be updated accordingly. The same would apply to `verify_user`.